frontend/package.json

```diff
@@ -26,8 +26,12 @@
     "test-gui-openshift": "yarn run test-suite --suite crud --params.openshift true",
     "test-gui": "yarn run test-suite --suite all",
     "test-suite": "ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
-    "test-cypress": "cd packages/integration-tests-cypress && cypress open --env openshift=true",
-    "test-cypress-headless": "cd packages/integration-tests-cypress && node --max-old-space-size=4096 ../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
+    "test-cypress": "yarn run test-cypress-common && yarn run test-cypress-olm",
```

Comment:
```suggestion
    "test-cypress": "cd packages/integration-tests-cypress && cypress open --env openshift=true",
```

Reply:
Thanks, this might clarify what we're eventually going to do:  https://issues.redhat.com/browse/CONSOLE-2426